cypress/e2e/3-our-e2e-tester/userTester.cy.js

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
+
+describe("Authentication flows", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:5173/login");
```

Comment:
Replace with [baseurl](https://docs.cypress.io/guides/references/best-practices#Setting-a-Global-baseUrl)
Read the docs.